src/ngraph_builder.cc

```diff
@@ -1736,6 +1743,8 @@ tf::Status Builder::TranslateGraph(const std::vector<tf::TensorShape>& inputs,
       TF_RETURN_IF_ERROR(GetInputNode(ng_op_map, op, 0, &ng_input));
 
       ng::AxisSet ng_axis_set;
+      // TODO: what is happening here? this is not using the const attr
```

Comment:
Strange, and a good catch. This attribute plays the same role as it does in `Sum` and `Mean`. Think we might have just missed this one when we made the ol' upgrade. If you want to fix in this PR that'd be swell, if not please just change this `TODO` to something like "should get reduction axis from op attribute".

FWIW, something like a `TranslateReductionOp`, analogous to `TranslateBinaryOp`, could unify those three ops here. Probably a separate PR.